phial/wrappers.py

```diff
@@ -143,3 +152,66 @@ def __init__(self,
 
     def __repr__(self) -> str:
         return "<Attachment in {0} >".format(self.channel)
+
+class MessageAttachment():
+    def __init__(self,
+                 fallback: Union[str, None] = None,
+                 color: Union[str, None] = None,
+                 author_link: Union[str, None] = None,
+                 author_icon: Union[str, None] = None,
+                 title: Union[str, None] = None,
+                 title_link: Union[str, None] = None,
+                 text: Union[str, None] = None,
+                 image_url: Union[str, None] = None,
+                 thumb_url: Union[str, None] = None,
+                 fields: Union[object, None] = None,
+                 footer: Union[str, None] = None,
+                 footer_icon: Union[str, None] = None) -> None:
+        self.fallback = fallback
+        self.color = color
+        self.author_link = author_link
+        self.author_icon = author_icon
+        self.title = title
+        self.title_link = title_link
+        self.text = text
+        self.image_url = image_url
+        self.thumb_url = thumb_url
+        self.fields = fields
+        self.footer = footer
+        self.footer_icon = footer_icon
+
```

Comment:
Could you add a `repr` function? This makes it easy to debug as when you print an instance of the class it will return a human-readable message as opposed to a memory location